src/scss/_common.scss

```diff
@@ -155,22 +155,50 @@
         position: relative;
         z-index: 1;
 
-        border: 1px solid $borderColor;
         box-sizing: border-box;
-        border-radius: 8px;
-        border-collapse: collapse;
+        border-collapse: separate;
         border-spacing: 0;
         background: $baseColor;
 
         thead, tr:nth-child(2n) {
             background-color: $tableRowBackgroundColor;
         }
 
+        tr:first-child td:first-child {
+            border-top-left-radius: $tableBorderRadius;
+        }
+
+        tr:first-child td:last-child {
+            border-top-right-radius: $tableBorderRadius;
+        }
+
+        tr:last-child td:first-child {
+            border-bottom-left-radius: $tableBorderRadius;
+        }
+
+        tr:last-child td:last-child {
+            border-bottom-right-radius: $tableBorderRadius;
+        }
+
+        tr:first-child td {
+            border-top-style: solid;
+        }
+
+        tr td:first-child {
+            border-top-style: solid;
```

Comment:
Final touch. Should change this line to `border-left: 1px solid $borderColor`. So we come from this:
<img width="160" alt="image" src="https://github.com/yandex-cloud/yfm-transform/assets/15200559/62a83520-d8b3-4950-8754-474270a01717">

to this:
<img width="169" alt="image" src="https://github.com/yandex-cloud/yfm-transform/assets/15200559/ae0c991a-b9c0-4b4b-8e89-3f3b70231cea">